resources/views/compare.blade.php

```diff
@@ -7,24 +7,35 @@
     <div class="container">
         <h1 class="mb-5 text-3xl font-bold">{{ __('Compare') }}</h1>
         <product-compare v-slot="compare">
-            <div class="w-full overflow-x-scroll">
-                <div class="flex py-4 z-20 border-b space-x-16">
-                    <div class="sticky left-0 w-1/5 flex-shrink-0 bg-white"></div>
-                    <a class="relative w-1/5 flex-shrink-0" v-for="item, key in compare.items" :href="item.product.url_key">
-                        <x-heroicon-c-x-mark class="absolute top-1 right-1 w-6 h-6" @click.prevent="compare.remove(item.uid)" />
-                        <img :src="item.product.image.url" :alt="item.product.image.name">
-                        <strong class="mt-5">
-                            @{{ item.product.name }}
-                        </strong>
-                    </a>
-                </div>
+            <div class="relative text-sm">
+                <div class="absolute left-0 top-0 bottom-5 w-1/2 sm:w-1/3 lg:w-1/4 xl:w-1/5 bg-white z-10 max-sm:hidden"></div>
+                <div class="overflow-hidden">
+                    <div class="overflow-x-auto overflow-y-hidden pb-5 -mt-px max-sm:-ml-3 -mr-px">
```

Comment:
On Windows scrollbar is shown by default. 
Do you want that to have this?

Reply:
Yes i do! We will add next and prev arrows in the future. So when that's the case i'll remove the scrollbar